openapi/components/requestBodies/userNotify.yaml

```diff
@@ -0,0 +1,32 @@
+description: A user
+content:
+  application/json:
+    schema:
+      type: object
+      required:
+        - message
+        - from
+      properties:
+        message:
+          type: string
+          required: true
+          description: |
+            Message title for the notification
```

Comment:
Think this should just be described as "The content of the notification message"..  Not sure 'title' is relevant as its not really a title.